setup.py

```diff
@@ -39,18 +39,30 @@
         "Twisted",
         "appdirs",
         "attrs",
-        "toga",
```

Comment:
so i think that the cocoa stuff doesn't work without a proper `Info.plist` anyway.... so maybe `.app` as the main OS X deployment target will have to do.